samples/bluetooth/direct_test_mode/src/fem/nrf21540.c

```diff
@@ -83,6 +83,12 @@ enum nrf21540_trx {
 	NRF21540_RX
 };
 
+enum nrf21540_state {
```

Comment:
Could you perhaps document which nRF21540 states map to these states for reference? E.g. `NRF21540_STATE_OFF` is _"Power-Down"_? Or are these "driver states" rather than "nRF21540 states"?

Reply:
Sure. I added a documentation for these state